### PR TITLE
Help text rendering using common code and std::string

### PR DESCRIPTION
**Pull request by AJenbo**

### CODE REVIEW

File: Source/help.cpp [hunk 1]
```diff
@@ -13,201 +15,157 @@
...
-static void DrawHelpLine(const CelOutputBuffer &out, int x, int y, char *text, uint16_t style)
-{
-	const int sx = x + 32 + PANEL_X;
-	const int sy = y * 12 + 44 + UI_OFFSET_Y;
-	DrawString(out, text, { sx, sy, 577, 0 }, style);
+	for (auto text : gszHelpText) {
+		strcpy(tempstr, _(text));
+
+		WordWrapGameString(tempstr, 577);
+		std::string paragraph = tempstr;
```
Comment (glebm) [inline Source/help.cpp:104]:
```suggestion
		std::string_view paragraph = tempstr;
```

Comment (AJenbo) [reply Source/help.cpp]: Fixed

File: Source/help.cpp [hunk 2]
```diff
@@ -13,201 +15,157 @@
...
+	for (auto text : gszHelpText) {
+		strcpy(tempstr, _(text));
+
+		WordWrapGameString(tempstr, 577);
+		const std::string_view paragraph = tempstr;
+
+		size_t previous = 0;
+		while (true) {
+			size_t next = paragraph.find("\n", previous);
+			HelpTextLines.push_back(paragraph.substr(previous, next));
```
Comment (glebm) [inline Source/help.cpp:109]: This can be `emplace_back` to call the `std::string(std::string_view)` constructor

Comment (AJenbo) [reply Source/help.cpp]: Ok :)

**Pull request merged**